x-pack/plugins/triggers_actions_ui/public/application/lib/check_action_type_enabled.tsx

```diff
@@ -85,6 +85,24 @@ const configurationCheckResult = {
   ),
 };
```

Comment:
I did a test run when running `trial` license and only enabling the index connector (via `plugins.actions.setEnabledConnectorTypes(['.index']);`) and I was still seeing the remaining connector types displayed as disabled in the UI (when they shouldn't be visible). This isn't the case when setting them from the yml setting (as designed).

Here's the scenarios and what I would expect:
- Disabled by license: Is grayed out in the UI
- Disable via yml setting: Is not visible in the UI
- Disable via registry: Is not visible in the UI
* When combining them, the "not visible" takes precedence over "grayed out".

We'll have to think and test the following combinations and ensure the outcomes are appropriate. It would be worth the effort to write automated tests for these eight scenarios to ensure we don't break this in the future.

Disabled by License | Disabled by YML | Disabled by Registry | UI State
-- | -- | -- | --
Y | Y | Y | Not visible
Y | Y | N | Not visible
Y | N | Y | Not visible
Y | N | N | Visible but Disabled
N | Y | Y | Not visible
N | Y | N | Not visible
N | N | Y | Not visible
N | N | N | Visible and Enabled (functional)

Reply:
I re-tested locally and all the scenarios work as expected now. I am not sure what happened for my computer to not get the latest changes but this morning it worked. Resolving..